program/shinyApp/R/data_selection/ui.R

```diff
@@ -12,22 +12,24 @@ data_selection_sidebar_panel <- sidebarPanel(
               width = "80%"
             )
         ),
+      HTML('<span style="font-size: 20px; font-weight: bold;">Upload</span> <small><a href="https://github.com/ICB-DCM/cOmicsArt/blob/main/UploadHelpcOmicsArt.xlsx" download>Download an example Excel Workbook to prepare data upload</a></small>'),
```

Comment:
Link does not work, and does not open in another page. I assume we need to put the xlsx into the www folder.

Reply:
```suggestion
      HTML('<span style="font-size: 20px; font-weight: bold;">Upload</span> <small><a href="https://github.com/ICB-DCM/cOmicsArt/blob/add_excel_workbook/UploadHelpcOmicsArt.xlsx" download>Download an example Excel Workbook to prepare data upload</a></small>'),
```
For now as main does not have the file. Can be changed at a later stage/changed to develop once we confirm it works

Reply:
I put it on purpose to the main link such that we do not have to change it twice - would stick with that for now?
It does work for the other link